js/src/modal.js

```diff
@@ -68,6 +68,7 @@ const Modal = (($) => {
     DATA_TOGGLE        : '[data-toggle="modal"]',
     DATA_DISMISS       : '[data-dismiss="modal"]',
     FIXED_CONTENT      : '.fixed-top, .fixed-bottom, .is-fixed, .sticky-top',
+    STICKY_CONTENT     : '.sticky-top',
```

Comment:
`.sticky-top' is in `FIXED_CONTENT` so I don't see the interest to add a new constant here :thinking:

Reply:
Because sticky elements also need the padding which was and still is applied to all fixed elements, in addition to the new margin, to have the correct position. However, I could change the references to STICKY_CONTENT to the string `.sticky-top` if you want.

Reply:
No that's fine I just wanted to understand your thinking 👍☺️